tests/envs/llvm/deterministic_actions_test.py

```diff
@@ -0,0 +1,44 @@
+# Copyright (c) Facebook, Inc. and its affiliates.
+#
+# This source code is licensed under the MIT license found in the
+# LICENSE file in the root directory of this source tree.
+"""Tests for action space determinism."""
+import hashlib
+
+import pytest
+
+from compiler_gym.envs import LlvmEnv
+from tests.test_main import main
+
+pytest_plugins = ["tests.envs.llvm.fixtures"]
+
+
+def test_deterministic_action(env: LlvmEnv, benchmark_name: str, action_name: str):
+    env.observation_space = "Ir"
+
+    checksums = set()
+    for i in range(1, 21):
```

Comment:
Why these constants?

Reply:
Consider passing the number of times in as param.

Reply:
Also consider doing some random steps first in case you just end up with no effect.
If doing rnd steps, consider having rnd seed param for repeatability.

Reply:
Good idea, done